content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -152,6 +152,21 @@ Once a pod reaches completion (has a `restartPolicy` of `Never` or `OnFailure` a
 The metrics are exposed at the HTTP endpoint `/metrics/resources` and require the same authorization as the `/metrics`
 endpoint on the scheduler. You must use the `--show-hidden-metrics-for-version=1.20` flag to expose these alpha stability metrics.
 
+## Disabling metrics
+
+You can explicitly turn off metrics via CLI flag `--disabled-metrics`. This may be desired if, for example, a metric is causing a performance problem. The input is a list of disabled metrics (i.e. `--disabled-metrics=metric1,metric2`).
+
+## Metric Cardinality Enforcement
+
+Metrics with unbounded dimensions could cause memory issues in the components they instrument. To solve this problem, we introduce a flag `--allow-label-value` to  dynamically configure an allow-list of label values for a metric.
+
+In alpha stage, the flag can only take in the allow-list value which is a [StringToString](https://pkg.go.dev/github.com/spf13/pflag#StringToStringVar) type argument from the CLI with the format 
```

Comment:
Avoid assuming that the reader can write Golang (many cluster operators don't / can't). For example:
```suggestion
This command line option takes a series of mappings. Each mapping is a of the format
`<metric_name>,<allowed_labels>` where the `<allowed_labels>` part is a comma-separated
list of acceptable label names.

The overall format looks like:
```

Reply:
Reworded.